test/eth/Web3Service.spec.js

```diff
@@ -97,11 +97,13 @@ test('fetch version info on connect', done => {
     .manager()
     .connect()
     .then(() => {
-      expect(web3.version().api).toMatch(/^([0-9]+\.)*[0-9]+$/);
-      expect(web3.version().node).toMatch(
+      expect(web3.version().api).toMatch(
+        /^([0-9]+\.)*[0-9]([-][b][e][t][a][.]([0-9])*)*$/
```

Comment:
The previous regex would only accept X.X.X version strings. Will now also accept X.X.X-beta.X.